docs/graphics/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow strict
+import AvatarGroup from './general/AvatarGroup.svg';
+
+export type IllustrationTypes = 'avatar-group';
```

Comment:
Rather than repeating the keys, check out [this utility type](https://flow.org/en/docs/types/utilities/#toc-keys)